bin/build_release

```diff
@@ -5,11 +5,6 @@ CURRENT_DIR=$("$(dirname "$0")/abspath")
 # xgo because it allows cross-compilation
 GORELEASER_IMAGE="cyberark/goreleaser:latest-xgo"
```

Comment:
whoa! are we still not able to use the official goreleaser image in our project?

Reply:
goreleaser doesn't provide an image with cross compilation capabilities :(

Reply:
The goreleaser main contributor is pretty responsive - maybe we can provide a PR?

Reply:
Might do this, yup